JapaneseFilter/sections/specific.txt

```diff
@@ -3640,7 +3635,7 @@ futabanet.jp##.r-ad-rectangle
 futabanet.jp##.m-ad-rectangle
 futabanet.jp##.ad-rectangle-right
 cyclowired.jp##.field-name-field-amazon-product1
-win10labo.info##.kizi-under-box
+win10labo.info.mtg60.com##.kizi-under-box
```

Comment:
Please correct the typo. It must be a comma before _mtg60.com_

Reply:
changed